packages/core/src/utils/format.ts

```diff
@@ -8,8 +8,8 @@ export const formatEta = (ms: number) => {
   const s = seconds - h * 3600 - m * 60;
 
   const hstr = h > 0 ? `${h}h ` : "";
-  const mstr = m > 0 || h > 0 ? `${m}m ` : "";
-  const sstr = s > 0 || m > 0 ? `${s}s` : "";
+  const mstr = m > 0 || h > 0 ? `${m < 10 ? "0" : ""}${m}m ` : "";
+  const sstr = s > 0 || m > 0 ? `${s < 10 ? "0" : ""}${s}s` : "";
```

Comment:
might only be worth prefixing if `m > 0`

```suggestion
  const sstr = s > 0 || m > 0 ? `${s < 10 && m > 0 ? "0" : ""}${s}s` : "";
```